macros/REST_Track_PlotAlphaAna.C

```diff
@@ -0,0 +1,341 @@
+//
+//
+//
+#include <TRestRun.h>
+#include <TRestTrackAlphaAnalysisProcess.h>
+#include <TRestDetectorReadout.h>
+#include <TRestTrackEvent.h>
+#include <TRestAnalysisTree.h>
+#include <TRestDetectorHitsEvent.h>
+#include <TRestRawSignalEvent.h>
+
+#include "dirent.h"
+
+TPad *DrawPulses ( TRestRawSignalEvent *rawSignalEvent, TRestDetectorReadout *fReadout) {
+
+  TRestDetectorReadoutPlane* plane = &(*fReadout)[0];
+
+  std::vector<Int_t> sXIDs;
+  std::vector<Int_t> sYIDs;
+
+  Int_t numberOfSignals = rawSignalEvent->GetNumberOfSignals();
+  Int_t readoutChannel = -1, readoutModule, planeID=0;
+  int maxX=0, maxY=0;
+    for (int i = 0; i < numberOfSignals; i++) {
+      TRestRawSignal* sgnl = rawSignalEvent->GetSignal(i);
+      Int_t signalID = sgnl->GetSignalID();
+      fReadout->GetPlaneModuleChannel(signalID, planeID, readoutModule, readoutChannel);
+      if(readoutChannel == -1)continue;
+      Double_t x = plane->GetX(readoutModule, readoutChannel);
+      Double_t y = plane->GetY(readoutModule, readoutChannel);
+        if (TMath::IsNaN(x)) {
+          sYIDs.emplace_back(signalID);
+        } else if (TMath::IsNaN(y)) {
+          sXIDs.emplace_back(signalID);
+        }
+    }
+
+  TPad *fPad = new TPad("RawSignalEvent", " ", 0, 0, 1, 1);
+  fPad->Divide(2);
+  fPad->Draw();
+  fPad->cd(1);
+  rawSignalEvent->DrawSignals(sXIDs);
+  fPad->Update();
+  fPad->cd(2);
+  rawSignalEvent->DrawSignals(sYIDs);
+  fPad->Update();
+
+ return fPad;
+
+}
+
+void DrawOriginEnd(TPad *pad, TGraph *originXZGr, TGraph *originYZGr,TGraph *endXZGr,TGraph *endYZGr){
+
+  pad->cd(1);
+  originXZGr->SetMarkerColor(kRed);
+  originXZGr->SetMarkerStyle(20);
+  originXZGr->Draw("LP");
+  endXZGr->SetMarkerColor(kBlack);
+  endXZGr->SetMarkerStyle(20);
+  endXZGr->Draw("LP");
+
+  pad->cd(2);
+  originYZGr->SetMarkerColor(kRed);
+  originYZGr->SetMarkerStyle(20);
+  originYZGr->Draw("LP");
+  endYZGr->SetMarkerColor(kBlack);
+  endYZGr->SetMarkerStyle(20);
+  endYZGr->Draw("LP");
+
+}
+
```

Comment:
Should be implemented at `TRestTools`? There is no documentation, what it is doing this method? Perhaps there is already a similar method at `TRestTools` doing what is doing this one?